tests/Http/Controllers/Api/Entries/View/External/ExportRoutes/ApiRateLimits/RateLimitsMediaExportLocalTest.php

```
@@ -0,0 +1,285 @@
+                $entryPayloads[$i]
+            );
+
+            // Validate entry row content against payload
+            $this->assertEntryRowAgainstPayload(
+                $entryRowBundle,
+                $entryPayloads[$i]
+            );
+
+            // Create a fake video file associated with the entry
+            $filename = $videoAnswers[$i]['answer'];
+            $videoContent = MediaGenerator::getFakeVideoContentBase64();
```

PHPMD (2.15.0)

Avoid using static access to class '\ec5\Libraries\Generators\MediaGenerator' in method 'test_rate_limit_api_media'. (Clean Code Rules)

(StaticAccess)